doc/docs/master/deploy-manage/deploy/prometheus/job_metrics.md

```diff
@@ -0,0 +1,24 @@
+Back to the [Prometheus set up page](../index).
+# List of job metrics exposed to Prometheus.

```

Comment:
Let’s change it to pipeline metrics. Instead of job metrics, I think it makes it clearer, even though you’re correct that these are metrics of jobs within a pipeline